environment/config.go

```diff
@@ -51,6 +55,40 @@ type Configuration struct {
 
 	// StaticConfiguration is used for running M3DB with a static config
 	Static *StaticConfiguration `yaml:"static"`
+
+	// Presence of a (etcd) server in this config denotes an embedded cluster
+	SeedNode *SeedNodeConfig `yaml:"seedNode"`
```

Comment:
nit: Because this is actually describes the seed nodes for the entire cluster, perhaps better to call it something like `seedNodes` with type `SeedNodesConfig`?